pkg/build/builder/common.go

```diff
@@ -81,3 +86,87 @@ func updateBuildRevision(c client.BuildInterface, build *api.Build, sourceInfo *
 		glog.Warningf("An error occurred saving build revision: %v", err)
 	}
 }
+
+func runPostCommitHook(dockerClient DockerClient, build *api.Build, strategyName, imageID string) error {
+	glog.Infof("Running post commit hook with image %s ...", imageID)
+	containerName := fmt.Sprintf("openshift_%s-build_%s_%s_post-commit_%08x", strategyName, build.Name, build.Namespace, rand.Uint32())
+	return runScriptInContainer(dockerClient, build.Spec.PostCommit.RunBash, imageID, containerName)
```

Comment:
According to a conversation with @bparees, there's also the problem that starting a pod would consume more of the users quota.